intro_to_r/intro_to_r.Rmd

```diff
@@ -58,14 +58,17 @@ library(oilabs)
 library(dplyr)
 library(ggplot2)
```

Comment:
Isn't `ggplot2` a dependency of `mosaic`? If that's the case, shouldn't we remove all references to it?